ldes-server-port-fetch-rest/pom.xml

```diff
@@ -50,6 +50,12 @@
             <groupId>io.swagger.core.v3</groupId>
             <artifactId>swagger-annotations-jakarta</artifactId>
         </dependency>
+        <dependency>
+            <groupId>be.vlaanderen.informatievlaanderen.vsds</groupId>
+            <artifactId>ldes-server-admin</artifactId>
+            <version>2.0.0-SNAPSHOT</version>
```

Comment:
version
I suppose you need this one, since EventStreamResponse is a shared object.